gengokit/handler/templates.go

```diff
@@ -48,3 +48,23 @@ type {{.PackageName}}Service struct{}
 	{{end}}
 {{- end}}
 `
+const hookTempl = `
+package handler
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func InterruptHandler(errc chan<- error) {
```

Comment:
@hasIan I really like the defer shutdown in main pattern you use there. I realize this PR is already merged, but I'll probably create another to implement it that way it's so nice!